pkg/storage/mongo/all_test.go

```diff
@@ -3,22 +3,19 @@ package mongo
 import (
 	"testing"
 
-	"github.com/gomods/athens/pkg/storage"
 	"github.com/stretchr/testify/suite"
 )
 
 type MongoTests struct {
 	suite.Suite
-	storage storage.Backend
 }
 
 func (d *MongoTests) SetupTest() {
-	store, err := NewStorage("mongodb://127.0.0.1:27017")
+	ms, err := newTestStore()
 
-	d.NoError(err)
+	d.Require().NoError(err)
 
-	store.s.DB(store.d).C(store.c).RemoveAll(nil)
-	d.storage = store
```

Comment:
How come we don't need to set `d.storage` anymore? That doesn't appear to be related to the PR.

Reply:
yes it is not related, we used that only to test creation of the storage but we don't use it anywhere so i removed it